api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -132,11 +132,9 @@ public void setUserService(UserService userService) {
     // TODO as part of Billing Project epic RW-1205: enable status checking for arbitrary project
     // until then: continue to check the user's free tier project status
     // because that is the only project we'll pass in here
-    if (!user.getFreeTierBillingProjectName().equals(billingProjectId)) {
-      throw new FailedPreconditionException(
-          "NOT YET IMPLEMENTED: Cannot list clusters using arbitrary project name");
-    }
-    if (user.getFreeTierBillingProjectStatusEnum() != BillingProjectStatus.READY) {
+    if (user.getFreeTierBillingProjectName() == null ||
```

Comment:
This was not quite my suggestion, logically here is what I'm proposing:

- billing project null --> continue (this supports a future where new users lack a free tier billing project)
- billing project != freeTier --> continue (this supports a future where an older user creates a new workspace with its own billing project)
- billing project == freeTier && notReady --> error (existing logic)

Reply:
Sorry - this was not clear to me.  I'm having trouble thinking about these future states and will follow up with you in a Googledoc.